src/components/ThemeProvider/__tests__/__snapshots__/Ant.test.tsx.snap

```diff
@@ -179,6 +186,12 @@ exports[`Generate Ant Theme generate ant theme from empty theme 1`] = `
       "defaultColor": undefined,
       "defaultHoverBg": "transparent",
     },
+    "Checkbox": {
+      "borderRadiusSM": NaN,
```

Comment:
is correct?

Reply:
kind of.. it's the snapshot for `Generate Ant Theme generate ant theme from empty theme 1`. 
It would be better to be 'undefined' probably, but checking it to be a valid number can be quite verbose here https://github.com/mia-platform/design-system/pull/701/files#diff-1d126b7a56770cd978a74dbfd3a9e76ecc379806a732261e03ab75f716f2fe49R31 .

Probably the right thing to do is to move the parse function at https://github.com/mia-platform/design-system/blob/main/src/components/ThemeProvider/Ant.tsx#L48 somewhere else and fix all the components theme all around in a separate pr.

Reply:
It's not that verbose actually, `NaN` is falsy, so you could just  add `|| undefined`.

Anyways, if we are sure that a NaN there is not source of issues, we could probably leave that there as is

Reply:
cool, i didn't knew it. I'm fixing this here then, but we have to remember to fix also the other components' themes (it's not a critical problem anyway leaving these as they are because the empty theme is used only in the snapshot)

Reply:
after thinking on this a second time... actually the borderRadiusSm could be set  to 0 intentionally so the "|| undefined" wouldn't work